hip/settings/base.py

```diff
@@ -256,6 +256,8 @@
     },
 }
 
+WAGTAILDOCS_EXTENSIONS = ["doc", "png", "jpg"]
```

Comment:
Don't we need `pdf` in here, or is that done automatically somehow? (haven't tested the PR out yet) :)

Reply:
🤦  saw that "doc" should have been "pdf", but forgot to commit it. Good 👁️